content/applications/sales/sales/invoicing/time_materials.rst

```diff
@@ -2,135 +2,352 @@
+   order **must** be marked as :guilabel:`Can be Expensed`, have an :guilabel:`Invoicing Policy` set
+   to :guilabel:`Delivered quantities` and the :guilabel:`At cost` option selected in the
+   :guilabel:`Re-Invoice Expenses` field on its product form.
+
+   .. image:: time_materials/product-form-settings-invoice-purchase.png
+      :align: center
+      :alt: Product settings for a purchase order to be invoiced on a sales order in Odoo.
+
+Then, select the appropriate :guilabel:`Analytic Distribution` associated with the sales order to
+which this purchase order is related. To do that, click the empty :guilabel:`Analytic Distribution`
+field to reveal an :guilabel:`Analytic` pop-up window.
+
+Then, from the :guilabel:`Departments` drop-down menu select the analytic distribution associated
+with the desired sales order to be invoiced for the purchase.
+
+.. image:: time_materials/analytic-drop-down-distribution.png
    :align: center
+   :alt: How to select the Analytic Distribution department from a purchase order in Odoo.
+
+Once all the information is entered in the :guilabel:`Products` tab of the purchase order, proceed
+to confirm the order by clicking :guilabel:`Confirm Order`. Then, click :guilabel:`Receive
+Products`.
+
+.. note::
+   If any serial/lot numbers must be entered before validation, click the :guilabel:`details` icon,
+   represented by four horizontal lines, located to the far-right of the product line to reveal a
+   :guilabel:`Detailed Operations` tab, in which the necessary :guilabel:`Lot/Serial Number(s)` and
+   :guilabel:`Done` quantity can be added.
 
-.. image:: time_materials/time_materials12.png
+When ready, click :guilabel:`Confirm` to confirm the data and return to the purchase order. Then,
+click :guilabel:`Validate` to validate the purchase order.
+
+Next, return to the purchase order, via the breadcrumb links at the top of the page, and click
+:guilabel:`Create Bill` to create a vendor bill that can be invoiced to the customer on the attached
+sales order.
+
+.. image:: time_materials/vendor-bill-draft.png
    :align: center
+   :alt: Vendor bill draft for a purchase order to be invoiced to a customer in Odoo.
+
+.. tip::
+   Make sure to enter a :guilabel:`Bill Date` on the :guilabel:`Vendor Bill Draft` before
+   confirming. If a :guilabel:`Bill Date` is *not* entered, an error window appears, requesting that
+   information to be entered before confirmation can occur.
 
-It will then be in the sales order and ready to be invoiced.
+Then, click :guilabel:`Confirm` to confirm the vendor bill, which is then automatically added to the
+sales order, where it can be invoiced directly to the customer attached to it.
 
-Invoice purchases
-=================
+Invoice purchase
+================
 
-The last thing you might need to add to the sale order is purchases made
-for it.
+To invoice a customer for a purchase on a sales order, navigate to the desired sales order in the
```

Comment:
For "add the purchase to the sales order," it would be great to link to the previous section "Add purchase to sales order". This will be great for someone skimming the doc.

It would be great if you could do this at the beginning of the other "Invoice..." sections as well ("Invoice time spent" and "Invoice expenses"). We want to make this doc readable/usable for people who are skimming and not following the entire flow of the doc
```suggestion
To invoice a customer for a purchase on a sales order, first add the purchase to the sales order, then navigate to the desired sales order in the
```